src/utilities/string-helper.ts

```diff
@@ -0,0 +1,7 @@
+export const getAllMatches = (pattern: RegExp, str: string): Array<RegExpExecArray> => {
```

Comment:
Does this have a benefit over https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/matchAll ?

Reply:
Nope, I've just had this problem before matchAll existed and didn't realize there was a better way 🤣 
I'll fix this in another pr, thanks for the heads up!